.github/workflows/nodejs.condo.ci.yml

```diff
@@ -106,6 +107,8 @@ jobs:
               - 'apps/dev-api/**'
               - 'apps/condo/domains/miniapp/**'
               - 'apps/condo/domains/user/**'
+            condorb:
+              - 'apps/condorb'
```

Comment:
You should probably trigger tests in condo changes (billing / market / acquiring domains at least), maybe even more. Maybe we should even always trigger this job, since it critical app I guess